### DISCUSSION TITLE
MissionControl 0.9.0

### DISCUSSION THREAD

**Discussion by ndeadly** [ndeadly/MissionControl#586] [General]
<p align="left">
<img alt="GitHub Releases" src="https://img.shields.io/github/downloads/ndeadly/MissionControl/v0.9.0/total">
<a href="https://discord.gg/gegfNZ5Ucz"><img alt="Discord Server" src="https://discordapp.com/api/guilds/905069757361971303/widget.png?style=shield"/></a>
</p>

This is the seventeenth official release of Mission Control.

This release adds support for firmware version 16.0.0, some new controllers, and fixes a few issues that popped up in recent releases. See below for the full changelog.

New users landing on this release page should first check out the readme on the [main project page](https://github.com/ndeadly/MissionControl#readme) for the official project documentation. There you can find installation and usage instructions along with an FAQ section that will answer most of your questions.

### Changelog:
- Updated libraries for 16.0.0 support.
- Added bluetooth exefs patches for 16.0.0.
- Added support for Sony Dualsense Edge controller.
- Added support for Razer Raiju Ultimate controller.
- Added support for Mad Catz L.Y.N.X 3 controller.
- Added support for AtGames Legends Pinball Arcade Control Panel extension. (Thanks @SK360)
- Added exefs patches for `btm` module to skip calls to CloseHidConnection when a controller doesn't respond properly to the SetTsi vendor command. This eliminates the need to manually flag certain controllers with the `settsi_disable.flag` file to stop them from being disconnected.
- Disabled incorrect handling of incoming NFC requests that caused games like Splatoon and Smash Bros to freeze or experience audio glitches when the game attempts to access amiibo data.
- Fixed regression in input handing for Wii Classic Controller/Classic Controller Pro extensions causing certain buttons not to be mapped.
- Fixed cross-platform building from source. (Thanks @cadewey)

<hr /><em>This discussion was created from the release <a href='https://github.com/ndeadly/MissionControl/releases/tag/v0.9.0'>MissionControl 0.9.0</a>.</em>

**Comment by Xadho** [comment]
is it just my switch or is my xbox one controller not connecting anymore?

**Reply by ndeadly** [reply]
Did you update it's firmware recently? Controller firmwares 5.xxxx.xx.x onwards don't work due to using Bluetooth LE

**Reply by Xadho** [reply]
yes I updated the firmware, but what your saying is, is that the Xbox controller doesn't work because of a Bluetooth issue?

**Reply by ndeadly** [reply]
The controller doesn't work because the firmware you updated to uses a different Bluetooth standard to the versions before it, and this standard isn't currently supported by Mission Control. You need to downgrade the controller firmware to continue using it. See [FAQ](https://github.com/ndeadly/MissionControl#frequently-asked-questions) for info.

**Reply by brolic925** [reply]
Any update on the stadia controller? Bluetooth has been unlocked

**Reply by ndeadly** [reply]
https://github.com/ndeadly/MissionControl/issues/568

**Reply by Jcderek-12** [reply]
So like I am trying to connect my ps4 controllers but it doesn't connect anymore, I don't know if I need Syscon still because they haven't updated to 16.0.0 so yeah my controllers just don't connect anyone know why?

**Reply by Masamune3210** [reply]
Make sure you are trying to connect them through Bluetooth, mission control doesn't support USB connections to controllers

**Comment by Jcderek-12** [comment]
Okay I'm gonna write another comment again because I really have no idea what is going on but before the 16.0.0 update my ps4 controllers would connect easily to the switch but ever since that update happened and we got a new mission control update I have never been able to connect it again like sure it used to all connect after a few minutes then it turn to an hour to connect then a few hours and now I can't even connect them and everyone else who uses it still connect their controllers in a few so idk what is the problem but if yall can help me out my discord is JCD#1227 I can really use the help porfavor
